src/Compilers/Core/MSBuildTask/Vbc.cs

```diff
@@ -651,9 +651,6 @@ protected override void LogEventsFromTextOutput(string singleLine, MessageImport
         /// Given a string, parses it to find out whether it's an error or warning and, if so,
         /// make sure it's validated properly.  
         /// </summary>
-        /// <comments>
-        /// INTERNAL FOR UNITTESTING ONLY
-        /// </comments>
```

Comment:
I deleted this because I don't see this method is for unit testing only. Please can someone confirm?

Reply:
It does seem to be unused other than above. Consider making this `private` instead, given that.

---
In reply to: [491003584](https://github.com/dotnet/roslyn/pull/47833#discussion_r491003584) [](ancestors = 491003584)